### PR TITLE
add selected theme to user informations endpoint

### PR DESCRIPTION
**Pull request by hyahiaoui-ext**
## Description
This PR intends to add selected theme property to the user infomations endpoint.

**Fixes** # MON-12164

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x (master)

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by kduret — COMMENTED** [review]
could you update api documentation in https://github.com/centreon/centreon/blob/MON-11932-dark-mode/doc/API/centreon-api-v22.04.yaml#L708

**Comment by hyahiaoui-ext** [comment]
> could you update api documentation in https://github.com/centreon/centreon/blob/MON-11932-dark-mode/doc/API/centreon-api-v22.04.yaml#L708

It's done :) !

**Review by kduret — APPROVED** [review]

File: src/Centreon/Domain/Contact/Contact.php [hunk 1]
```diff
@@ -175,6 +175,11 @@ class Contact implements UserInterface, ContactInterface
      */
     private $isOneClickExportEnabled = false;
 
+    /**
+     * @var string
```
Comment (callapa) [inline src/Centreon/Domain/Contact/Contact.php:179]:
```suggestion
     * @var string|null
```

File: src/Centreon/Domain/Contact/Contact.php [hunk 2]
```diff
@@ -658,4 +663,26 @@ public function getUserIdentifier(): string
...
+    public function setTheme(string $theme): self
+    {
+        $this->theme = $theme;
+        return $this;
+    }
+
+    /**
+     * Get user current theme.
+     *
+     * @return string
```
Comment (callapa) [inline src/Centreon/Domain/Contact/Contact.php:682]:
```suggestion
     * @return string|null
```

File: src/Centreon/Domain/Contact/Contact.php [hunk 3]
```diff
@@ -658,4 +663,26 @@ public function getUserIdentifier(): string
...
+     */
+    public function getTheme(): string
```
Comment (callapa) [inline src/Centreon/Domain/Contact/Contact.php:684]:
```suggestion
    public function getTheme(): ?string
```

**Review by callapa — CHANGES_REQUESTED** [review]

**Review by callapa — APPROVED** [review]

**Pull request merged**